### PR TITLE
Update users/userId/:userId to users/?id=abc api url

### LINKED ISSUE #336: Update the path `users/userId/:userId` to `users/?id=abc` in dashboard site
**This need to be deployed along with issue** https://github.com/Real-Dev-Squad/website-backend/issues/966.

### PR DESCRIPTION
**Pull request by sahsisunny**
# [Merge this after it's 👇 done ] 

- [x] https://github.com/Real-Dev-Squad/website-backend/pull/1064

# Update users/userId/:userId to users/?id=abc api url

### What is the change?

- `profile/utils.js` - Change `users/userId/:userId to users/?id=abc` api url

### Is Development Tested?

- [x] Yes
- [ ] No

### CODE REVIEW

File: profile/utils.js [hunk 1]
```diff
@@ -331,7 +331,7 @@ async function getSelfUser() {
 }
 
 async function getUser(userId) {
-  const userResponse = await fetch(`${API_BASE_URL}/users/userId/${userId}`, {
+  const userResponse = await fetch(`${API_BASE_URL}/users/?id=${userId}`, {
```
Comment (vikhyat187) [inline profile/utils.js:334]: We can remove the extra slash after `users`.

**Review by vikhyat187 — CHANGES_REQUESTED** [review]

File: profile/utils.js (hunk 1 above)
Comment (sahsisunny) [reply profile/utils.js]: Done

**Review by endeavourmonk — APPROVED** [review]
Looks good to me.

**Comment by vikhyat187** [comment]
@sahsisunny DO NOT MERGE this before this PR https://github.com/Real-Dev-Squad/website-backend/pull/1064, once @endeavourmonk has pushed this to staging can you verify in local and after the backend gets into prod, you can deploy this.

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**